network/p2p/middleware.go

```diff
@@ -50,7 +50,7 @@ const (
 
 	// default maximum time to wait for a default unicast request to complete
 	// assuming at least a 1mb/sec connection
-	DefaultUnicastTimeout = 2 * time.Second
+	DefaultUnicastTimeout = 10 * time.Second
```

Comment:
are we sure we want to move up to 10 seconds?

Reply:
maybe you are right - changed it back to 2 seconds. The Ping latency for both BT and T-systems is high (120ms+) but T-systems seem to be doing ok, so may be 2 seconds default is ok. But at least, we have a setting to play with in case we start seeing the timeouts.